addons/common/XEH_preStart.sqf

```diff
@@ -1,3 +1,13 @@
 #include "script_component.hpp"
 
 #include "XEH_PREP.hpp"
+
+// These functions are used for the ace arsenal, one of which are called in preStart (ace_arsenal_fnc_scanConfig)
+// Cache for FUNC(getAddon)
+uiNamespace setVariable [QGVAR(addonCache), createHashMap];
```

Comment:
Is it guaranteed, that `common`'s preStart will run before `arsenal`'s? If not, I will have to add these cache definitions to the functions themselves.

Reply:
if common is in requiredAddons of arsenal (which it has to be) yes.